planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/include/autoware/behavior_path_goal_planner_module/decision_state.hpp

```diff
@@ -62,11 +62,10 @@ class PathDecisionStateController
 
   PathDecisionState get_current_state() const { return current_state_; }
 
-  PathDecisionState get_prev_state() const { return prev_state_; }
-
 private:
+  rclcpp::Logger logger_;
+
   PathDecisionState current_state_{};
-  PathDecisionState prev_state_{};
 
   /**
    * @brief update current state and save old current state to prev state
```

Comment:
Only `prev_state_.stamp` was used for saving the time when DECIDING state started.
So I deleted `prev_state_` and save `deciding_start_time` instead.